packages/dom/src/dom/is-form-element.js

```diff
@@ -7,14 +7,28 @@ import isInputOrTextArea from './is-input-or-text-area';
  *
  * Detects if element is a form element.
  *
- * @param {Element} element The element to check.
+ * @param {Element} element                The element to check.
+ * @param {boolean} includeContentEditable Rather or not to include elements with contentEditable attribute.
  *
  * @return {boolean} True if form element and false otherwise.
  */
-export default function isFormElement( element ) {
+export default function isFormElement(
+	element,
+	includeContentEditable = false
+) {
 	const { tagName } = element;
 	const checkForInputTextarea = isInputOrTextArea( element );
+	let checkContentEditable = false;
```

Comment:
I added this because some blocks such as the table block have contentEditable instead of a real form element.